src/module/actor/actor-sheet.js

```diff
@@ -534,7 +534,19 @@ export class OseActorSheet extends ActorSheet {
     });
 
     html.find(".item-delete").click((event) => {
-      this._removeItemFromActor(event);
+      const item = this._getItemFromActor(event);
+      const itemData = item?.system;
+
+      if (item.type === "container" && itemData.itemIds.length > 0) {
```

Comment:
`itemdata = item?.system` is something we'd do if we expect to get here without having a valid item to delete. If so, we might want to add similar checks to the `if` statement here.

Reply:
Yes, I realize that I declared `itemData` only as a helper in the if-statement, so it makes no sense to keep given your suggested change above  - fixed in latest commit!